subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -46,6 +46,11 @@ struct lll_conn_iso_stream {
 	uint8_t active:1;           /* 1 if CIS LLL is active */
 	uint8_t datapath_ready_rx:1;/* 1 if datapath for RX is ready */
 
+#if !defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+	/* lazy at CIS active */
+	uint16_t lazy_active;
```

Comment:
What unit is this? Just based on the struct field name, it is unclear what this does or what the value range is.

Reply:
updated comment.